common-lib/common/signal/signal.libsonnet

```diff
@@ -31,8 +32,8 @@ local stub = import './stub.libsonnet';
   // DEPRECATED. Use unmarshallJsonMulti instead.
```

Comment:
Do we have an issue for tracking this?

Reply:
Not really. I think it should be safe to remove for now, i'll check this repo for uses.